### PR TITLE
[coordinator] Ensure rewrite middleware doesn't set the body on a GET

### PR DESCRIPTION
**Pull request by nbroyles**
**What this PR does / why we need it**:

Setting the body on a GET request can sometimes cause
"http: invalid Read on closed Body" errors within the Golang
HTTP client. As such, ensure that if processing a GET request
the body is not set.

**Special notes for your reviewer**:

**Does this PR introduce a user-facing and/or backwards incompatible change?**:

```release-note

```

**Does this PR require updating code package or user-facing documentation?**:

```documentation-note

```

### CODE REVIEW

**Review by wesleyk — APPROVED** [review]
LGTM, worth leaving a note in code?

File: src/query/api/v1/middleware/rewrite.go [hunk 1]
```diff
@@ -101,6 +101,10 @@ func rewriteRangeDuration(
 			r.Form.Del(endParam)
 		}
 
+		if r.Method == "GET" {
```
Comment (wesleyk) [inline src/query/api/v1/middleware/rewrite.go:104]: should this be before the above check?

Comment (nbroyles) [reply src/query/api/v1/middleware/rewrite.go:104]: Nah, we still want to purge from `r.Form` the modifications we made solely for the sake of time parsing.

**Comment by nbroyles** [comment]
> LGTM, worth leaving a note in code?

Eh, I figured the commit message is probably enough for anyone investigating

**Pull request merged**